controllers/common/index.js

```diff
@@ -92,7 +95,9 @@ function renderFlexibleContentChild(req, res, entry) {
     res.render(path.resolve(__dirname, './views/flexible-content-page'), {
         breadcrumbs: res.locals.breadcrumbs.concat({ label: res.locals.title }),
         // @TODO: Can we rename API responses to consistently name this flexibleContent?
-        content: { flexibleContent: entry.content },
+        entry: Object.assign(entry, {
```

Comment:
This is a little awkward. For some reason the API uses `flexibleContent` for the main flexible content pages and `content` for the occasional child pages that support flexible content. We should come back and update the API to use the same name but otherwise this works.